### PR TITLE
#0: remove data movement ops related to silu in SD

### PR DESCRIPTION
**Pull request by mtatsumiTT**
it passes post-commit pipeline: https://github.com/tenstorrent-metal/tt-metal/actions/runs/8457977137

### CODE REVIEW

File: models/experimental/functional_stable_diffusion/tt2/ttnn_functional_resnetblock2d.py [hunk 1]
```diff
@@ -456,14 +454,7 @@ def __call__(
                 memory_config=self.second_gn_expected_input_sharded_memory_config,
                 core_grid=self.second_group_norm_core_grid,
             )
-        hidden_states = ttnn.to_memory_config(hidden_states, ttnn.L1_MEMORY_CONFIG)
-        hidden_states = ttnn.reshape(
```
Comment (AleksKnezevic) [inline models/experimental/functional_stable_diffusion/tt2/ttnn_functional_resnetblock2d.py:460]: What's the input shape here? Don't we need this reshape?

File: models/experimental/functional_stable_diffusion/tt2/ttnn_functional_unet_2d_condition_model.py [hunk 2]
```diff
@@ -580,18 +580,7 @@ def __call__(
                 memory_config=self.gn_expected_input_sharded_memory_config,
                 core_grid=self.group_norm_core_grid,
             )
-        sample = ttnn.to_memory_config(sample, ttnn.L1_MEMORY_CONFIG)
-        sample = ttnn.reshape(
-            sample,
-            (
```
Comment (AleksKnezevic) [inline models/experimental/functional_stable_diffusion/tt2/ttnn_functional_unet_2d_condition_model.py:586]: Similar as above, what's the input shape here, the conv expects 1,1,bhw,c

File: models/experimental/functional_stable_diffusion/tt2/ttnn_functional_resnetblock2d.py (hunk 1 above)
Comment (mtatsumiTT) [reply models/experimental/functional_stable_diffusion/tt2/ttnn_functional_resnetblock2d.py:460]: I double-checked it with running the test again, it's (1, 1, bhw, c)

File: models/experimental/functional_stable_diffusion/tt2/ttnn_functional_unet_2d_condition_model.py (hunk 2 above)
Comment (mtatsumiTT) [reply models/experimental/functional_stable_diffusion/tt2/ttnn_functional_unet_2d_condition_model.py:586]: right, it is (b, 1, hw, c). I'll add the reshape back.

Comment (AleksKnezevic) [reply models/experimental/functional_stable_diffusion/tt2/ttnn_functional_unet_2d_condition_model.py:586]: I'm a little baffled that the test passes without it... did you see any PCC drop?

Comment (mtatsumiTT) [reply models/experimental/functional_stable_diffusion/tt2/ttnn_functional_unet_2d_condition_model.py:586]: no I didn't notice obvious drop, the unet test passes.

**Review by AleksKnezevic — APPROVED** [review]

**Review by arakhmati — APPROVED** [review]

**Pull request merged**